test/torchaudio_unittest/batch_consistency_test.py

```diff
@@ -283,3 +283,14 @@ def test_batch_Vol(self):
         # Batch then transform
         computed = torchaudio.transforms.Vol(gain=1.1)(waveform.repeat(3, 1, 1))
         self.assertEqual(computed, expected)
+
+    def test_batch_spectral_centroid(self):
+        test_filepath = common_utils.get_asset_path('steam-train-whistle-daniel_simon.wav')
```

Comment:
Like in `torchscript_compatibility_test`, can you use random or pseudo waveform?
The use of I/O functions is a legacy and problematic practice.